libs/debugger/src/debugger.rs

```diff
@@ -201,6 +207,47 @@ macro_rules! mprint {
 impl<'a> Debugger<'a> {
     /// Create a new debugger and attach to `pid`
     pub fn attach(pid: u32) -> Debugger<'a> {
+        Debugger::attach_internal(pid, false)
+    }
+
+    /// Create a new process argv[0], with arguments argv[1..] and attach to it
+    pub fn spawn_proc(argv: &[String], follow_fork: bool) -> Debugger<'a> {

```

Comment:
This function should take a `&[&str]` rather than a `&[String]`, as it's just a superset of all strings that could be passed.